doc/tech/roadmap.md

```diff
@@ -78,24 +90,21 @@ The various workstreams in CNTT are
 
 The workstream dependency relationship in simple terms, [Reference Conformance](../ref_cert) verifies and tests the [Reference Implementation](../ref_impl) which follows the requirements and architecture defined in the CNTT [Reference Architecture](../ref_arch) and Reference Architecture describes the high level system components and its interactions by adhering to the requirements and expectations set by the CNTT [Reference Model](../ref_model) which sets the standards for infrastructure abstraction, compliance and verification. 
 
-For the standard release stabilisation, the pattern in each release follows RM must be the advanced version of RA. Similarly RA must be the advanced version of RI and simultaneously for RC.
-
-For the standard release stabilisation, each release follows the pattern of one workstream is functioning ahead of another workstream in the sequence of Reference Model, Reference Architecture, Reference Implementation and Reference Conformance. For eg. In Baldy release, RM3.0 inputs will feed into RA2.0 and likewise as shown in the Figure 3.
+For the standard release stabilisation, On each release, All documents that are related to each other will have the same **main** version number as shown in the Figure 3.
 
 There are two different tracks in CNTT
 - Virtualized workloads, deployed on OpenStack
 - Containerized workloads, deployed on Kubernetes
 
 Each track follows the industry driven standards in the Reference Model as depicted in the below diagram.
 
-
-<p align="center"><img src="./figures/CNTT_Stable_3.0.png" alt="CNTT Stable 3.0 Portrait" title="CNTT Stable 3.0 Portrait
+<p align="center"><img src="./figures/tech_ws_dependencies.png" alt="CNTT WS Dependencies" title="CNTT WS Dependencies
 " width="100%"/>
- <p align="center"><b>Figure 3:</b> CNTT Stable 3.0 Portrait</p>
+ <p align="center"><b>Figure 3:</b> CNTT WS Dependencies</p>
```

Comment:
Figure 3, version numbers of RA, RI, RC etc are different from those in Figure 2. Figure 2 has the correct version numbers.

Reply:
fixed